### ISSUE TITLE
Add Sphinx Support

### ISSUE THREAD

**Issue by ekultails** [ekultails/rootpages#2]
Sphinx is one of the top standards for building various types of documentation formats (ePub, HTML, PDF, etc.) based on RST or Markdown source code. This can even be used to create a full book. Root Pages should start using Sphinx for it's build process.

http://www.sphinx-doc.org/en/stable/contents.html

- [x] Convert documents from Markdown to ReStructuredText.
- [x] Run "sphinx-quickstart" to setup the build configuration files.
- [x] Remove old build scripts.
- [x] Update the README file on how to use Sphinx to build documentation.
- [x] Correct the headers to only have the title of each document (for example, "Ansible") be the `#1` header. All other headers should be bumped down by one level to be nested under the document's title's name within a table of contents.

**Comment by ekultails** [comment]
Creating and maintaining tables in the ReStructuredText format is very difficult because the format has to be very exact and matching the length and spacing of the text within it. More research should be done on how open source projects normally manage these tables.

**Comment by ekultails** [comment]
Adding Sphinx support is actually incredible easy thanks to the [Sphinx tutorial and the sphinx-quickstart tool](http://www.sphinx-doc.org/en/stable/tutorial.html).

There is [support for Markdown in Sphinx](http://www.sphinx-doc.org/en/stable/markdown.html). I tested it out and found that the Markdown parsing is lacking. The "CommonMark" library did not render links, code blocks, and tables correctly. It might not be fully compatible with GitHub or Pandoc styles of Markdown; one of the pitfalls of lacking a standard format.

Due to this, I will be putting Root Pages on a temporary "code freeze" while I convert all of the documents into RST (using Pandoc and then manually editing them to ensure the formats and styling is correct). The only problem I found with RST was that it is complicated, and time consuming, to create a grid table. Luckily, tools such as [Atom with the rst-snippets plugin](https://atom.io/packages/rst-snippets) and [Vim with the vim-rst-tables plugin](https://github.com/nvie/vim-rst-tables) help make creating and maintaining tables in RST an easy process. I will eventually note these tools in the README.md file.
